NewBank/newbank/server/NewBank.java

```diff
@@ -52,4 +57,40 @@ private String showMyAccounts(CustomerID customer) {
 		return (customers.get(customer.getKey())).accountsToString();
 	}
 
+	private String move(CustomerID customer, String[] splitRequest){
+		// check there are 4 things in the request
+		if (splitRequest.length != 4){
+			return "FAIL";
+		}
+		// check both bank accounts exist for the customer
+		ArrayList<Account> customerAccounts = customers.get(customer.getKey()).getAccountList();
+		double amount = Double.parseDouble(splitRequest[1]);
+		//get the account objects (instead of the accountName)
+		ArrayList<Account> moveAccounts = new ArrayList<>();
+		for (int i = 0; i < customerAccounts.size(); i++) {
+			if (customerAccounts.get(i).getName().equals(splitRequest[2])) {
+				moveAccounts.add(customerAccounts.get(i));
+			}
+		}
+		for (int i = 0; i < customerAccounts.size(); i++){
+			if (customerAccounts.get(i).getName().equals(splitRequest[3])) {
+				moveAccounts.add(customerAccounts.get(i));
```

Comment:
This functionality could probably be moved to a separate function in the Customer class. Something like

public Account getAccount(String name)

this could return null if the account does not exist (or some other way to notify that account does not exist). 
Note that those two for loops (line 70 and 75) do almost exactly the same thing which might suggest it could be a function